Tests/Functional/MappingToElasticaTest.php

```diff
@@ -21,11 +21,13 @@ class MappingToElasticaTest extends WebTestCase
     public function testResetIndexAddsMappings()
     {
         $client = $this->createClient(array('test_case' => 'Basic'));
-        $resetter = $this->getResetter($client);
-        $resetter->resetIndex('index');
 
         $type = $this->getType($client);
         $mapping = $type->getMapping();
+        $resetter = $this->getResetter($client);
```

Comment:
I've done a PR against your PR (PR-inception!) to address these issues: https://github.com/emgiezet/FOSElasticaBundle/pull/1